.jenkins/pytorch/build.sh

```diff
@@ -61,7 +61,7 @@ fi
 WERROR=1 python setup.py install
 
 # Add the test binaries so that they won't be git clean'ed away
-git add -f build/bin
+git add -f build/bin build/lib
```

Comment:
Wait, why do you need this directory? On my build it weighs 700M, a good 30% of the overall 2.6G that's the build directory. And if we installed PyTorch, we should already have these libraries skulking around elsewhere. Is the rpath not working or something?

Reply:
> It maintains the behavior from before this diff that the tests are run out of their build directory

But no one actually cares about that. The actual functional requirements are:

1. The tests in CI need to find the libraries, somehow. (E.g., you could just use `LD_LIBRARY_PATH` to fix the path situation)
2. When I, a random developer, run tests from the build directory, they ought to work (without me having to `LD_LIBRARY_PATH` explicitly)
3. We should not be shipping multiple copies of the same libraries and binaries in the Docker image when we push from CPU to GPU. (But if you want to show that this doesn't actually make a difference, e.g., in time to push, then we can drop this requirement.)